src/stratus-gwc/src/main/resources/geowebcache-servlet.xml

```diff
@@ -18,14 +18,14 @@
   <import resource="geowebcache-virtualearth-context.xml"/>
   <import resource="geowebcache-wmsservice-context.xml"/>
   <import resource="geowebcache-wmtsservice-context.xml"/>
-
   <import resource="geowebcache-diskquota-context.xml"/>
     
   <!--
     This mappings are different from the standalone gwc ones in that they prepend the /gwc prefix to the context so it
     ends up being, for example, /geoserver/gwc/*
   -->
-  <bean class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
+  <bean class="org.springframework.context.support.PropertySourcesPlaceholderConfigurer">
+  <!--<bean class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">-->
```

Comment:
Some explanaition might be nice here? Or just delete the commented out bean?

Reply:
Ya, spring just refactored the packages.  Removed the commented line.